aldryn_search/utils.py

```diff
@@ -134,19 +136,14 @@ def strip_tags(value):
     strip tags. If value isn't valid, just return value since there is
     no tags to strip.
     """
-    # borrowed from aldryn-search
-
     if isinstance(value, six.string_types):
         value = value.strip()
 
-        if not value:
```

Comment:
if value is an empty string, lxml won't raise a ParserError, instead it will raise a different exception. That's why I had done the check with if